src/inventory-brand/api/routes/index.ts

```diff
@@ -0,0 +1,11 @@
+import CategoryValidation from "../validations";
+import { baseRouter, baseValidation } from "../../../inventory-shared/api";
+
+import { BrandController } from "../controllers/brand";
+import { AuthMiddleware } from "../../../inventory-auth/middlewares/authMiddleware";
+
+const {POST, router} = baseRouter();
+
+POST("/add-brand", [baseValidation(CategoryValidation.brand), AuthMiddleware.baseAuthToken, AuthMiddleware.IsUserMiddleware, BrandController.brand]);
```

Comment:
this should be "/" not "/add-brand" as create operations for POST are commonly named